tests/DataFixtures/ORM/PromotionsFixture.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\DataFixtures\ORM;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\Promotion;
+use DateTime;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class PromotionsFixture extends AbstractFixture implements DependentFixtureInterface
+{
+    private $manager;
+
+    public function getDependencies()
+    {
+        return [
+            ImagesFixture::class,
+            MongrelsFixture::Class,
+        ];
+    }
+
+    public function load(ObjectManager $manager)
+    {
+        $this->manager = $manager;
+
+        $this->buildPromotion(
+            'p000000h',
+            'active promotion of CoreEntity',
+            $this->getReference('b00swyx1'), // promotion of Core entity - series 1
+            1, // weight
+            true,
+            new DateTime('1900-01-01'),
```

Comment:
These are all datetimes, so should we specify the time too? `new DateTime('1900-01-01 00:00:00')`